bndtools.core/src/bndtools/command/BndrunFilesParameterValues.java

```diff
@@ -33,25 +31,21 @@
 
 	@Override
 	public Map getParameterValues() {
-		Workspace ws = Central.getWorkspaceIfPresent();
+		if (bndrunFilesMap != null) {
```

Comment:
This seems like a perfect place for a memoizier to get lazy initialization and it will properly handle the synchronization for at-most-once initialization.

Reply:
yeah we should, I'll try my hand at it.

Reply:
check it now!